android/src/main/kotlin/com/u0x48lab/body_detection/BodyDetectionPlugin.kt

```diff
@@ -90,13 +92,19 @@ class BodyDetectionPlugin: FlutterPlugin, MethodChannel.MethodCallHandler, Event
         cameraSession = null
         result.success(true)
       }
+      "switchCamera" -> {
+        val isFront = call.argument("lensFacing") as String?
+        lensFacing =
+          if (isFront?.equals("FRONT") == true) CameraSelector.LENS_FACING_BACK else CameraSelector.LENS_FACING_BACK
```

Comment:
For if, there should be CameraSelector.LENS_FACING_FRONT. Since both condition have same statement you cannot toggle back to front camera after switching it to the back camera for the first time. Please update this in your commit. I found your commit very helpful for my project. This was the only issue I found. ThankYou:) Happy Coding